src/Prjkt/Component/Repofuck/Repofuck.php

```diff
@@ -307,13 +307,17 @@ protected function map(array $inserts, array $keys = [], $entity = null) : Model
 			throw new EntityNotDefined;
 		}
 
+		$fillable = $entity->getFillable();
+		
 		foreach($inserts as $key => $val)
 		{
 			if ( count($keys) > 0 && ! in_array($keys, $key) ) {
 				break;
 			}
 
-			$entity->{$key} = $val;
+			if (array_key_exists($key, $fillable)){
```

Comment:
I think `$keys` and and `$keys` should switch place in `in_array`.

Is the `$key` the needle and `$keys` is the haystack?